metrics/web/docs/features/storybook/01_create_metrics_storybook.md

```diff
@@ -0,0 +1,196 @@
+
+Pros:
+
+- Allows managing themes.
+- Allows to group widgets by their types.
+- Contains a list of inputs to change widgets' appearance.
+
+Cons:
+
+- Provides only a plain UI.
+- On the preview screen, users cannot select the viewport size.
+- Does not have a docs tab that supports MDX (or similar) with the documentation about the widget.
+- Does not have a search across components functionality.
+- It's not so popular (has only a few stars on GitHub).
+- Requires updating a Dart version to 2.12.0 or higher.
+
+4. [Storybook_flutter](https://pub.dev/packages/storybook_flutter)
+
+A cross-platform storybook for showcasing widgets. It works on all platforms supported by Flutter.
+
+Pros:
+
+- Allows managing themes.
+- Allows creating a custom Storybook (e.g. UI).
+- Allows approximating how a particular widget looks and performs on different devices (requires additional related package).
+
+Cons:
+
+- Provides only a plain UI.
+- Does not have a docs tab that supports MDX (or similar) with the documentation about the widget.
+- Does not have a search across components functionality.
+- Requires updating a Dart version to 2.12.0 or higher.
+- It's not so popular (has only a few stars on GitHub).
+
+#### Manually create a Storybook from scratch
+
+Another solution is to provide a completely new Flutter project with the [required functionality](#requirements).
+
+Pros:
+
+- Allows creating a custom Storybook with all required features.
+
+Cons:
+
+- Requires designing a new UI for the Metrics storybook.
+- Takes extra time to realize the new UI.
+
+#### Conclusion
+
+The described above packages do not suit us for several reasons:
+
+- They are rather libraries to showcasing widgets with an ability to change their appearance, but the storybook is a bit more than that. A storybook contains documentation about widgets, information about the possible parameters they can accept, search across their functionality, and more.
+
+- The management of themes in these packages is reduced only to the possibility of switching between the dark and the light themes, as well as the presence of a toggle theme button, the behavior of which we cannot control. Even though the packages allow managing themes, it becomes a complex task to integrate the Metrics theme into the storybook. The described above packages do not allow to provide the theme data for all the Metric widgets. See [Metrics Theme guidelines](https://github.com/Flank/flank-dashboard/blob/master/metrics/web/docs/03_widget_structure_organization.md#metrics-theme-guidelines) to get more metrics theme implementation details.
+
+- The UI in these packages is, also very basic, without thoughtful design and UX.
+
+Summing up the consideration of approaches and given the above explanations, the best solution would be to create this library from [scratch](#manually-create-a-storybook-from-scratch).
+
```

Comment:
It seems like the following sections are missing: 
    - [Prototyping](https://github.com/Flank/flank-dashboard/blob/master/docs/19_feature_design_template.md#prototyping)
    - [System modeling](https://github.com/Flank/flank-dashboard/blob/master/docs/19_feature_design_template.md#system-modeling)